docs/reference/metadata.md

```diff
@@ -170,42 +170,102 @@ while others can only be accessed via the [API][].
 
 ### Indentation Options
 
-Indentation options control aspects of the auto-indentation mechanism.
+Indentation options control aspects of the auto-indentation mechanism;  
+Use regex to match a single line, and make change to the indent-level of the lines following the matched line;  
+The `decreaseIndentPattern` option also change the indent-level of the matched line itself.  
 
 `increaseIndentPattern`
 : *Regex.*
   If it matches on the current line,
-  the next line will be indented one level further.
+  lines after current line will be indented one level further.
 
-  ```xml
-  <key>increaseIndentPattern</key>
-  <string>insert regex here</string>
+*format*  
```

Comment:
I suspect starting on column 0 will cancel the definition body and mess up the formatting.

Following the examples we have above, I don't deem the "format" to even be necessary as the example further down should be enough to show how it's used.